node/config/settings/__init__.py

```diff
@@ -6,7 +6,9 @@
 
 from node.core.utils.pytest import is_pytest_running
 
-BASE_DIR = Path(__file__).resolve().parent.parent.parent.parent
+ROOT_DIR = Path(__file__).resolve().parent.parent.parent.parent
+
+ENV_CONFIG_PATH = os.path.join(ROOT_DIR, '.env')
```

Comment:
not a good idea to relate `.env` file to source code. if the source code is install in virtualenv the .env file will appear somewhere in the virtualenv folder